### ISSUE TITLE
Microsoft Edge fails the "Iframe dispatch should not affect Sizzle, see jQuery #13936" test

### ISSUE THREAD

**Issue by mzgol** [jquery/sizzle#339]
Microsoft Edge (from the newest Windows 10 build 10130) fails the "Iframe dispatch should not affect jQuery (#13936)" test with an error:
```js
{
  "description": "Permission denied",
  "number": -2146828218
}
```
The test suite it fails a lot more tests that follow after this one but most likely it's just this one leaves the browser in a bad state.

Originally reported at https://github.com/jquery/jquery/issues/2390.

Screenshot:
![screen shot 2015-06-15 at 18 09 08](https://cloud.githubusercontent.com/assets/1758366/8164494/b6c6fcac-1389-11e5-9ea9-286973cf5f97.png)

**Comment by mzgol** [comment]
For people following, there has been a discussion about that at https://github.com/jquery/sizzle/commit/28f4c67c508719cc76e2075dce0e4cbe0e325919.

**Comment by mzgol** [comment]
Fixed by #341 (I forgot to amend the commit message. :/)
